third-party/dnd-wiki/classes/class-weapon-master.xml

```diff
@@ -0,0 +1,454 @@
+		<description>
+			<p>Starting at 6th level, your strikes become so graceful, precise and swift, that you can easily target exposed vital organs of a creature before they have enough time to react. Your weapon attack score a critical hit on a roll of a 19 or 20.</p>
+		</description>
+		<sheet>
+			<description>Your weapon attack score a critical hit on a roll of a 19 or 20.</description>
+		</sheet>
+	</element>
+	<element name="Flawless Stance" type="Class Feature" source="D&amp;D Wiki" id="ID_WIKI_CLASS_FEATURE_WEAPON_MASTER_FLAWLESS_STANCE">
+		<description>
+			<p>At 9th level, you become extremely hard to fight due to your uncanny stance. When a creature you can see or hear attacks you, you can use your reaction to make attacks from that creature roll with disadvantage when attacking you. If the creature misses an attack against you, in the same reaction you can counter attack, making one melee attack against that creature. You can use this feature a number of times equal to your Dexterity modifier(minimum 1) and regain uses of this feature when you finish a long rest.</p>
+		</description>
+		<sheet>
+			<description>When a creature you can see or hear attacks you, you can use your reaction to make attacks from that creature roll with disadvantage when attacking you. If the creature misses an attack against you, in the same reaction you can counter attack, making one melee attack against that creature. You can use this feature a number of times equal to your Dexterity modifier(minimum 1) and regain uses of this feature when you finish a long rest.</description>
+		</sheet>
+	</element>
+	<element name="Adrenaline" type="Class Feature" source="D&amp;D Wiki" id="ID_WIKI_CLASS_FEATURE_WEAPON_MASTER_ADRENALINE">
+		<description>
+			<p>Beginning at 13th level, when you enter combat your body instantly responds to your battle urges releasing a heavy dose of adrenaline into your system. After rolling your Initiative roll, you gain temporary hit points equal to your weapon master level.</p>
+		</description>
+		<sheet>
+			<description>After rolling your Initiative roll, you gain temporary hit points equal to your weapon master level.</description>
+		</sheet>
+	</element>
+	<element name="Imposing Fighter" type="Class Feature" source="D&amp;D Wiki" id="ID_WIKI_CLASS_FEATURE_WEAPON_MASTER_IMPOSING_FIGHTER">
+		<description>
+			<p>At 14th level, you gain proficiency in the Intimidation skill, if you already have proficiency in it, you can double your proficiency bonus.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<grant type="Grants" id="ID_GRANTS_PROFICIENCY_SKILL_INTIMIDATION" requirements="!ID_PROFICIENCY_SKILL_INTIMIDATION"/> 
+			<grant type="Grants" id="ID_GRANTS_PROFICIENCY_SKILL_INTIMIDATION_DOUBLE" requirements="ID_PROFICIENCY_SKILL_INTIMIDATION"/>
+		</rules>
+	</element>
+	<element name="Intimidation Proficiency" type="Grants" source="Core" id="ID_GRANTS_PROFICIENCY_SKILL_INTIMIDATION">
```

Comment:
you can use the internal workaround element here now

Reply:
Done, also used workaround for Hunt Master Archetype Feature.